completion/services.py

```diff
@@ -62,3 +74,27 @@ def get_completions(self, candidates):
             if candidate not in completions:
                 completions[candidate] = 0.0
         return completions
+
```

Comment:
On Line 72, you have to change `block.block_key` to `block.full_block_key`.  See: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/completion/services.py#L70

Reply:
Good catch. It's a miss on my part. Done